examples/external_request/with_harvester.rb

```diff
@@ -0,0 +1,82 @@
+require 'net/http'
+require 'bundler'
+Bundler.require
+
+require "new_relic/telemetry_sdk"
+
+unless ENV["API_KEY"]
+  raise "No License key supplied.  Export API_KEY environment variable!" 
+end
+
+def setup_buffer_harvesting common_attributes = {test_attribute: 'example'}
+  @harvester = NewRelic::TelemetrySdk::Harvester.new 
+  @span_client = NewRelic::TelemetrySdk::SpanClient.new
+  @buffer = NewRelic::TelemetrySdk::Buffer.new common_attributes
```

Comment:
It might be nice to add a comment stating that common attributes will be added to all spans recorded to the buffer.